readme.md

```diff
@@ -56,7 +56,21 @@ The thing you want to open. Can be a URL, file, or executable.
 
 Opens in the default app for the file type. Eg. URLs opens in your default browser.
 
-#### app
+#### options
+
+Type: `object`
+
+An optional options hash.
+
+##### options.wait
+
+Type: `boolean`
+
+Defaults to `true` and means that opn will wait calling the callback (if one is given) until the app have been closed again. If set to `false` the callback will be called immediately after the child process have spawned.
+
+Note that waiting is only possible if the app name is either `darwin` or `win32`.
```

Comment:
I was confused by `app name`. I guess you meant OS name.

But no, it will wait on Linux too, it's just default, so there's no flag.